lib/workers/repository/update/pr/index.spec.ts

```diff
@@ -6,7 +6,7 @@ import {
   platform,
 } from '../../../../../test/util';
 import { PlatformId } from '../../../../constants';
-import type { Pr } from '../../../../modules/platform/types';
+import type { Pr } from '../../../../modules/platform';
```

Comment:
I think this was an unintended change. 😉

Reply:
Yeah. Should not do this. Just `/types` not required there